src/interfaces/operation.ts

```diff
@@ -205,20 +205,7 @@ export enum OperationPostInvokeConfirmType {
 /**
  * The action that will be performed after the user confirms it
  */
-export interface OperationPostInvokeConfirm {
-    /**
-     * Type of postInvokeConfirm action
-     */
-    type: OperationPostInvokeConfirmType | string
-    /**
-     * Title for modal
-     */
-    message: string
-    /**
-     * Additional text for modal
-     */
-    messageContent?: string
-}
+export type OperationPostInvokeConfirm = OperationModalInvokeConfirm
```

Comment:
I think we should reverse it: the key part is that we've received post invoke for confirmation, and then we can decide how to handle it. By default we handle it with modal window, but client application might be able to handle this type of post actions via popups, notifications or any other way.

So I think we should keep `OperationPostInvokeConfirm` and deprecate through alias `OperationModalInvokeConfirm`.

Reply:
resolved